tools/onboarder/README.md

```diff
@@ -0,0 +1,10 @@
+## The Onboarder Script
+This script will generate an Ethereum keystore file in json format and will load the account with testnet ether as well as respective tokens. Run
+```
+python onboarder.py
```

Comment:
Does this actually work? I only tried with the binary.

Reply:
@agatsoh ?

Reply:
Hello Jacob I tested it, and it works !!!